ert_shared/ensemble_evaluator/prefect_ensemble/prefect_ensemble.py

```diff
@@ -218,7 +218,8 @@ def get_flow(self, ee_id, dispatch_url, input_files, real_range):
                         output_to_res.get(input, []) for input in step.get("inputs", [])
                     ]
                     stage_task = UnixStep(
-                        resources=[input_files[iens]] + step["resources"],
+                        resources=[input_files[iens]]
+                        + self.store_resources(step["resources"]),
```

Comment:
By doing the store_resources here, aren't we doing it over for every time a task runs?

Reply:
We are not doing it when the task runs we are doing it when the task is set up.